app/actions/bonds.ts

```diff
@@ -344,9 +344,11 @@ export const redeemTransaction = async (params: {
   bond: Bond;
   provider: providers.JsonRpcProvider;
   onStatus: OnStatusHandler;
+  shouldAutostake: boolean;
 }) => {
   try {
-    const autostake = false;
+    // update this line if we change prop to required
+    const autostake = params.shouldAutostake;
```

Comment:
ha now that my ugly hardcoded var is gone we can just pass `params.shouldAutostake` directly to the contract function below

Reply:
oops, fixed.